amida-tech/indaba-client
# Feature/INBA-498 Wizard Delete Group

#### What's this PR do?
Delete groups from the wizard with no prompt

#### Related JIRA tickets:
[INBA-498](https://jira.amida-tech.com/browse/INBA-498)

#### How should this be manually tested?
1. Start the project wizard
1. Add user groups
1. Delete them
1. Check that they are deleted from the backend. Previously they were removed locally and disappeared but no call was made to the backend.

#### Any background context you want to provide?
Groups with users in them will fail on the backend delete (see [INBA-499](https://jira.amida-tech.com/browse/INBA-499))

This branch sits on #249, review that first for a smaller changeset

#### Screenshots (if appropriate):
